src/sec_certs/dataset/cc.py

```diff
@@ -164,38 +169,10 @@ def certificates_txt_dir(self) -> Path:
         """
         return self.certificates_dir / "txt"
 
-    @property
-    def pp_dataset_path(self) -> Path:
-        """
-        Returns a path to the dataset of Protection Profiles
-        """
-        return self.auxiliary_datasets_dir / "pp_dataset.json"
-
-    @property
-    def mu_dataset_dir(self) -> Path:
-        """
-        Returns directory that holds dataset of maintenance updates
-        """
-        return self.auxiliary_datasets_dir / "maintenances"
-
-    @property
-    def mu_dataset_path(self) -> Path:
-        """
-        Returns a path to the dataset of maintenance updates
-        """
-        return self.mu_dataset_dir / "maintenance_updates.json"
-
```

Comment:
I need a way to get to these somehow, when handling the updates on the site. Can I do it with the new API? Iterate over aux_handlers and get their paths and detect based on type?

Reply:
They form a dictionary and each child of `AuxiliaryDatasetHandler` must implement:
- `root_dir` attribute, where the dataset files are stored.
- `dset_path` attribute, where the final json is stored. 

Example, in the case of `CPEDataset` you can:

```python
cpe_dset = cc_dset.aux_handlers[CPEDatasetHandler].dset
cpe_dset_json_path = cc_dset.aux_handlers[CPEDatasetHandler].dset_path
```

Reply:
Makes sense.